### PR TITLE
Bug (Editor view and inspectors of plugins are created twice) + code …

### PR DESCRIPTION
**Pull request by Roman-Rostovtsev**
…formatting + made default constructor private since the pattern Singleton is applied

### CODE REVIEW

File: soapui/src/main/java/com/eviware/soapui/support/editor/registry/InspectorRegistry.java [hunk 1]
```diff
@@ -32,14 +32,13 @@
     private static InspectorRegistry instance;
     private List<InspectorFactory> factories = new ArrayList<InspectorFactory>();
```
Comment (spindelmanne) [inline soapui/src/main/java/com/eviware/soapui/support/editor/registry/InspectorRegistry.java]: While you're here, introduce the diamond operator to remove some noise: new ArrayList<>()

Comment (Roman-Rostovtsev) [inline soapui/src/main/java/com/eviware/soapui/support/editor/registry/InspectorRegistry.java]: Ok, I've removed type names

File: soapui/src/main/java/com/eviware/soapui/support/editor/registry/EditorViewFactoryRegistry.java [hunk 2]
```diff
@@ -29,14 +29,14 @@
 import java.util.List;
 
 /**
- * Registry of availabel XmlViews
+ * Registry of avaliable XmlViews
```
Comment (spindelmanne) [inline soapui/src/main/java/com/eviware/soapui/support/editor/registry/EditorViewFactoryRegistry.java]: Still misspelled! :-) Should be "available"

**Comment by IlyaAvdeev** [comment]
@spindelmanne I totally forgot about this PR. Feel free to merge if it's ok.

**Comment by spindelmanne** [comment]
Has it been verified by a tester?

**Comment by IlyaAvdeev** [comment]
@spindelmanne of course we'll check it, but it will be done after merging this PR (need to create complete build of the product)

**Pull request merged**